[2024-10-16 00:00:01,981.981] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:00:06,641.641] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:00:06,644.644] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:00:09,334.334] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:00:09,642.642] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:00:14,317.317] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:00:14,321.321] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:00:17,703.703] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 00:00:17] "GET /conversions?query=你能简单介绍下什么是LLM么、 HTTP/1.1" 200 -
[2024-10-16 00:02:45,344.344] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 00:02:45] "GET /conversions HTTP/1.1" 200 -
[2024-10-16 00:05:03,719.719] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 00:05:03] "GET /conversions?query=你能简单介绍下什么是LLM么、 HTTP/1.1" 200 -
[2024-10-16 00:05:26,219.219] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 00:05:26] "GET /conversions?query=你好你是？ HTTP/1.1" 200 -
[2024-10-16 00:06:26,475.475] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:06:26,847.847] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:06:31,599.599] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:06:31,603.603] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:06:46,371.371] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 00:06:46] "GET /conversions?query=写一篇关于春天的诗词 HTTP/1.1" 200 -
[2024-10-16 00:06:47,752.752] http.py -> _register_error_handler line:66 [ERROR]: An error occurred: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
Traceback (most recent call last):
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 844, in dispatch_request
    self.raise_routing_exception(req)
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 453, in raise_routing_exception
    raise request.routing_exception  # type: ignore[misc]
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\ctx.py", line 362, in match_request
    result = self.url_adapter.match(return_rule=True)  # type: ignore
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\werkzeug\routing\map.py", line 624, in match
    raise NotFound() from None
werkzeug.exceptions.NotFound: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
[2024-10-16 00:06:47,782.782] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 00:06:47] "[35m[1mGET /favicon.ico HTTP/1.1[0m" 500 -
[2024-10-16 00:07:13,611.611] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 00:07:13] "GET /conversions?query=你能简单介绍下什么是LLM么、 HTTP/1.1" 200 -
[2024-10-16 00:08:59,916.916] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:09:00,244.244] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:09:05,415.415] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:09:05,417.417] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:09:23,044.044] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:09:23,390.390] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:09:28,259.259] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:09:28,262.262] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:11:46,538.538] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:11:46,848.848] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:14:05,041.041] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:14:09,920.920] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:14:09,923.923] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:14:25,277.277] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:14:25,610.610] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:14:30,490.490] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:14:30,495.495] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:14:48,162.162] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:14:48,504.504] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:14:53,259.259] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:14:53,262.262] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:15:44,634.634] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:15:44,933.933] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:15:49,599.599] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:15:49,602.602] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:15:53,368.368] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:15:53,693.693] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:15:58,428.428] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:15:58,431.431] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:16:24,154.154] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:16:24,481.481] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:16:29,176.176] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:16:29,181.181] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:16:35,331.331] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:16:35,640.640] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:16:47,992.992] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:16:57,116.116] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:17:01,692.692] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:17:01,696.696] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:17:10,158.158] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:17:10,462.462] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:17:15,095.095] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:17:15,097.097] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:17:44,278.278] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:17:44,592.592] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:17:49,448.448] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:17:49,452.452] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:17:52,064.064] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:17:52,389.389] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:17:57,227.227] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:17:57,230.230] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:17:58,705.705] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:17:59,021.021] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:18:03,710.710] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:18:03,712.712] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:18:29,431.431] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:18:29,751.751] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:18:34,539.539] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:18:34,543.543] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:18:44,122.122] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:18:44,444.444] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:18:49,281.281] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:18:49,284.284] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:19:40,476.476] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:19:40,802.802] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:19:45,436.436] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:19:45,439.439] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:19:51,453.453] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:19:51,851.851] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:19:56,552.552] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:19:56,555.555] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:20:45,414.414] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:20:45,798.798] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:20:50,437.437] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:20:50,440.440] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:21:42,049.049] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:21:42,389.389] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:21:47,217.217] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:21:47,220.220] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:22:30,659.659] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:22:31,009.009] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:22:35,821.821] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:22:35,824.824] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:22:47,002.002] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:22:47,321.321] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:23:13,675.675] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:23:18,348.348] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:23:18,352.352] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:24:16,876.876] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:24:17,218.218] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:24:22,203.203] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:24:22,206.206] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:24:23,000.000] base.py -> _exec_single_context line:1903 [INFO]: select pg_catalog.version()
[2024-10-16 00:24:23,000.000] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:24:23,001.001] base.py -> _exec_single_context line:1903 [INFO]: select current_schema()
[2024-10-16 00:24:23,001.001] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:24:23,001.001] base.py -> _exec_single_context line:1903 [INFO]: show standard_conforming_strings
[2024-10-16 00:24:23,002.002] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:24:23,002.002] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 00:24:23,004.004] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 00:24:23,004.004] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00018s] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 00:24:23,009.009] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 00:24:23,009.009] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00019s] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 00:24:23,010.010] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 00:24:49,282.282] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:24:49,626.626] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:24:54,490.490] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:24:54,493.493] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:24:54,711.711] base.py -> _exec_single_context line:1903 [INFO]: select pg_catalog.version()
[2024-10-16 00:24:54,711.711] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:24:54,712.712] base.py -> _exec_single_context line:1903 [INFO]: select current_schema()
[2024-10-16 00:24:54,712.712] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:24:54,713.713] base.py -> _exec_single_context line:1903 [INFO]: show standard_conforming_strings
[2024-10-16 00:24:54,713.713] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:24:54,714.714] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 00:24:54,715.715] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 00:24:54,715.715] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00016s] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 00:24:54,719.719] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 00:24:54,719.719] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00018s] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 00:24:54,720.720] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 00:24:55,908.908] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 00:24:55] "GET /conversions?query=你好，你是？ HTTP/1.1" 200 -
[2024-10-16 00:25:10,874.874] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 00:25:10,875.875] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 00:25:10,875.875] base.py -> _exec_single_context line:1908 [INFO]: [cached since 16.16s ago] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 00:25:10,876.876] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 00:25:10,876.876] base.py -> _exec_single_context line:1908 [INFO]: [cached since 16.16s ago] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 00:25:10,877.877] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 00:25:11,711.711] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 00:25:11] "GET /conversions?query=广州的天气怎么样 HTTP/1.1" 200 -
[2024-10-16 00:25:44,177.177] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 00:25:44,178.178] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 00:25:44,178.178] base.py -> _exec_single_context line:1908 [INFO]: [cached since 49.46s ago] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 00:25:44,179.179] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 00:25:44,179.179] base.py -> _exec_single_context line:1908 [INFO]: [cached since 49.46s ago] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 00:25:44,180.180] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 00:25:45,080.080] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 00:25:45] "GET /conversions?query=你好，你是？ HTTP/1.1" 200 -
[2024-10-16 00:27:30,680.680] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:27:31,009.009] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:27:35,752.752] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:27:35,754.754] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:27:43,319.319] base.py -> _exec_single_context line:1903 [INFO]: select pg_catalog.version()
[2024-10-16 00:27:43,319.319] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:27:43,320.320] base.py -> _exec_single_context line:1903 [INFO]: select current_schema()
[2024-10-16 00:27:43,320.320] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:27:43,321.321] base.py -> _exec_single_context line:1903 [INFO]: show standard_conforming_strings
[2024-10-16 00:27:43,321.321] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:27:43,322.322] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 00:27:43,324.324] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 00:27:43,324.324] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00030s] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 00:27:43,328.328] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 00:27:43,328.328] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00022s] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 00:27:43,329.329] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 00:27:44,886.886] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 00:27:44] "GET /conversions?query=广州的天气怎么样？ HTTP/1.1" 200 -
[2024-10-16 00:32:25,279.279] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 00:32:25,279.279] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 00:32:25,279.279] base.py -> _exec_single_context line:1908 [INFO]: [cached since 282s ago] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 00:32:25,281.281] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 00:32:25,281.281] base.py -> _exec_single_context line:1908 [INFO]: [cached since 282s ago] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 00:32:25,281.281] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 00:32:26,377.377] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 00:32:26] "GET /conversions?query=广州的天气怎么样？ HTTP/1.1" 200 -
[2024-10-16 00:36:35,065.065] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:36:35,394.394] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:36:40,092.092] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:36:40,095.095] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:36:47,491.491] base.py -> _exec_single_context line:1903 [INFO]: select pg_catalog.version()
[2024-10-16 00:36:47,491.491] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:36:47,492.492] base.py -> _exec_single_context line:1903 [INFO]: select current_schema()
[2024-10-16 00:36:47,493.493] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:36:47,493.493] base.py -> _exec_single_context line:1903 [INFO]: show standard_conforming_strings
[2024-10-16 00:36:47,493.493] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:36:47,494.494] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 00:36:47,497.497] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 00:36:47,497.497] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00025s] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 00:36:47,501.501] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 00:36:47,501.501] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00027s] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 00:36:47,503.503] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 00:36:49,046.046] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 00:36:49] "GET /conversions?query=帮我查询下广州的天气预报 HTTP/1.1" 200 -
[2024-10-16 00:39:15,479.479] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:39:15,794.794] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:39:20,811.811] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:39:20,814.814] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:39:22,280.280] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:39:22,612.612] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:39:27,329.329] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:39:27,332.332] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:39:36,862.862] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:39:37,188.188] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:39:41,930.930] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:39:41,933.933] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:39:42,141.141] base.py -> _exec_single_context line:1903 [INFO]: select pg_catalog.version()
[2024-10-16 00:39:42,141.141] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:39:42,142.142] base.py -> _exec_single_context line:1903 [INFO]: select current_schema()
[2024-10-16 00:39:42,142.142] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:39:42,143.143] base.py -> _exec_single_context line:1903 [INFO]: show standard_conforming_strings
[2024-10-16 00:39:42,143.143] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:39:42,144.144] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 00:39:42,145.145] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 00:39:42,145.145] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00013s] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 00:39:42,148.148] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 00:39:42,148.148] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00017s] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 00:39:42,150.150] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 00:39:56,219.219] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:39:56,553.553] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:40:01,078.078] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:40:01,081.081] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:40:03,681.681] base.py -> _exec_single_context line:1903 [INFO]: select pg_catalog.version()
[2024-10-16 00:40:03,681.681] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:40:03,682.682] base.py -> _exec_single_context line:1903 [INFO]: select current_schema()
[2024-10-16 00:40:03,682.682] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:40:03,683.683] base.py -> _exec_single_context line:1903 [INFO]: show standard_conforming_strings
[2024-10-16 00:40:03,683.683] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:40:03,684.684] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 00:40:03,686.686] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 00:40:03,686.686] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00019s] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 00:40:03,689.689] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 00:40:03,689.689] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00020s] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 00:40:03,689.689] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 00:40:41,881.881] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 00:40:41,881.881] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 00:40:41,881.881] base.py -> _exec_single_context line:1908 [INFO]: [cached since 38.2s ago] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 00:40:41,883.883] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 00:40:41,883.883] base.py -> _exec_single_context line:1908 [INFO]: [cached since 38.19s ago] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 00:40:41,884.884] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 00:44:25,436.436] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:44:25,760.760] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:44:30,425.425] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:44:30,428.428] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:44:30,661.661] base.py -> _exec_single_context line:1903 [INFO]: select pg_catalog.version()
[2024-10-16 00:44:30,661.661] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:44:30,662.662] base.py -> _exec_single_context line:1903 [INFO]: select current_schema()
[2024-10-16 00:44:30,662.662] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:44:30,663.663] base.py -> _exec_single_context line:1903 [INFO]: show standard_conforming_strings
[2024-10-16 00:44:30,663.663] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:44:30,663.663] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 00:44:30,665.665] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 00:44:30,665.665] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00018s] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 00:44:30,669.669] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 00:44:30,669.669] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00029s] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 00:44:30,670.670] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 00:46:14,975.975] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:46:15,333.333] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:48:01,501.501] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:48:24,263.263] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:48:33,317.317] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:48:38,413.413] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:48:38,416.416] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:48:41,232.232] base.py -> _exec_single_context line:1903 [INFO]: select pg_catalog.version()
[2024-10-16 00:48:41,232.232] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:48:41,233.233] base.py -> _exec_single_context line:1903 [INFO]: select current_schema()
[2024-10-16 00:48:41,233.233] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:48:41,234.234] base.py -> _exec_single_context line:1903 [INFO]: show standard_conforming_strings
[2024-10-16 00:48:41,234.234] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:48:41,234.234] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 00:48:41,236.236] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 00:48:41,236.236] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00015s] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 00:48:41,239.239] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 00:48:41,239.239] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00020s] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 00:48:41,241.241] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 00:49:06,645.645] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:49:06,976.976] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:49:11,620.620] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:49:11,622.622] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:49:33,144.144] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:49:33,464.464] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:49:38,479.479] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:49:38,482.482] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:49:41,830.830] base.py -> _exec_single_context line:1903 [INFO]: select pg_catalog.version()
[2024-10-16 00:49:41,830.830] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:49:41,832.832] base.py -> _exec_single_context line:1903 [INFO]: select current_schema()
[2024-10-16 00:49:41,832.832] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:49:41,832.832] base.py -> _exec_single_context line:1903 [INFO]: show standard_conforming_strings
[2024-10-16 00:49:41,832.832] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:49:41,833.833] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 00:49:41,834.834] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 00:49:41,835.835] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00016s] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 00:49:41,838.838] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 00:49:41,838.838] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00018s] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 00:49:41,839.839] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 00:50:18,285.285] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 00:50:18,607.607] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 00:50:23,205.205] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 00:50:23,208.208] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 00:50:30,239.239] base.py -> _exec_single_context line:1903 [INFO]: select pg_catalog.version()
[2024-10-16 00:50:30,239.239] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:50:30,240.240] base.py -> _exec_single_context line:1903 [INFO]: select current_schema()
[2024-10-16 00:50:30,241.241] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:50:30,241.241] base.py -> _exec_single_context line:1903 [INFO]: show standard_conforming_strings
[2024-10-16 00:50:30,241.241] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 00:50:30,242.242] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 00:50:30,245.245] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 00:50:30,245.245] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00023s] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 00:50:30,249.249] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 00:50:30,249.249] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00032s] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 00:50:30,251.251] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 00:50:32,451.451] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 00:50:32] "GET /conversions?query=广州的天气怎么样？ HTTP/1.1" 200 -
[2024-10-16 08:13:48,723.723] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 08:13:49,085.085] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 08:13:54,746.746] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 08:13:54,749.749] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 08:14:00,875.875] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 08:14:01,222.222] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 08:14:06,132.132] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 08:14:06,134.134] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 08:14:07,676.676] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 08:14:08,003.003] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 08:14:12,743.743] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 08:14:18,495.495] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 08:14:30,061.061] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 08:14:35,172.172] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 08:14:35,175.175] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 08:14:40,118.118] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 08:14:40,485.485] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 08:14:45,543.543] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 08:14:45,546.546] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 08:14:59,788.788] base.py -> _exec_single_context line:1903 [INFO]: select pg_catalog.version()
[2024-10-16 08:14:59,788.788] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:14:59,789.789] base.py -> _exec_single_context line:1903 [INFO]: select current_schema()
[2024-10-16 08:14:59,789.789] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:14:59,790.790] base.py -> _exec_single_context line:1903 [INFO]: show standard_conforming_strings
[2024-10-16 08:14:59,790.790] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:14:59,791.791] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 08:14:59,794.794] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 08:14:59,794.794] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00029s] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 08:14:59,798.798] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 08:14:59,798.798] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00027s] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 08:14:59,800.800] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 08:15:01,911.911] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 08:15:01] "GET /conversions?query=广州的天气怎么样？ HTTP/1.1" 200 -
[2024-10-16 08:15:33,368.368] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 08:15:33,794.794] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 08:15:38,572.572] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 08:15:38,575.575] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 08:15:38,798.798] base.py -> _exec_single_context line:1903 [INFO]: select pg_catalog.version()
[2024-10-16 08:15:38,799.799] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:15:38,799.799] base.py -> _exec_single_context line:1903 [INFO]: select current_schema()
[2024-10-16 08:15:38,800.800] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:15:38,800.800] base.py -> _exec_single_context line:1903 [INFO]: show standard_conforming_strings
[2024-10-16 08:15:38,800.800] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:15:38,801.801] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 08:15:38,803.803] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 08:15:38,803.803] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00024s] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 08:15:38,807.807] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 08:15:38,807.807] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00022s] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 08:15:38,808.808] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 08:15:40,257.257] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 08:15:40] "GET /conversions?query=广州的天气怎么样？ HTTP/1.1" 200 -
[2024-10-16 08:15:57,707.707] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 08:15:58,114.114] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 08:16:02,948.948] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 08:16:02,949.949] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 08:16:12,929.929] base.py -> _exec_single_context line:1903 [INFO]: select pg_catalog.version()
[2024-10-16 08:16:12,929.929] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:16:12,930.930] base.py -> _exec_single_context line:1903 [INFO]: select current_schema()
[2024-10-16 08:16:12,931.931] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:16:12,931.931] base.py -> _exec_single_context line:1903 [INFO]: show standard_conforming_strings
[2024-10-16 08:16:12,931.931] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:16:12,931.931] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 08:16:12,933.933] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 08:16:12,933.933] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00019s] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 08:16:12,936.936] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 08:16:12,936.936] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00018s] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 08:16:12,937.937] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 08:16:14,503.503] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 08:16:14] "GET /conversions?query=广州的天气怎么样？ HTTP/1.1" 200 -
[2024-10-16 08:17:36,218.218] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 08:17:36,686.686] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 08:17:41,496.496] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 08:17:41,499.499] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 08:18:55,029.029] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 08:18:55,378.378] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 08:19:00,102.102] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 08:19:00,105.105] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 08:19:03,927.927] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 08:19:04,270.270] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 08:19:09,060.060] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 08:19:09,062.062] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 08:20:01,827.827] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 08:20:02,177.177] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 08:20:06,637.637] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 08:20:06,639.639] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 08:20:08,099.099] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 08:20:08,493.493] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 08:20:12,931.931] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 08:20:12,934.934] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 08:20:25,938.938] base.py -> _exec_single_context line:1903 [INFO]: select pg_catalog.version()
[2024-10-16 08:20:25,938.938] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:20:25,940.940] base.py -> _exec_single_context line:1903 [INFO]: select current_schema()
[2024-10-16 08:20:25,940.940] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:20:25,941.941] base.py -> _exec_single_context line:1903 [INFO]: show standard_conforming_strings
[2024-10-16 08:20:25,941.941] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:20:25,942.942] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 08:20:25,945.945] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 08:20:25,945.945] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00031s] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 08:20:25,949.949] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 08:20:25,950.950] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00028s] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 08:20:25,951.951] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 08:20:27,795.795] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 08:20:27] "GET /conversions?query=广州的天气怎么样？ HTTP/1.1" 200 -
[2024-10-16 08:21:51,899.899] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 08:21:52,373.373] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 08:21:58,882.882] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 08:21:58,885.885] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 08:21:59,101.101] base.py -> _exec_single_context line:1903 [INFO]: select pg_catalog.version()
[2024-10-16 08:21:59,101.101] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:21:59,102.102] base.py -> _exec_single_context line:1903 [INFO]: select current_schema()
[2024-10-16 08:21:59,102.102] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:21:59,103.103] base.py -> _exec_single_context line:1903 [INFO]: show standard_conforming_strings
[2024-10-16 08:21:59,103.103] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:21:59,103.103] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 08:21:59,105.105] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 08:21:59,105.105] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00017s] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 08:21:59,109.109] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 08:21:59,109.109] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00019s] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 08:21:59,110.110] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 08:22:00,466.466] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 08:22:00] "GET /conversions?query=广州的天气怎么样？ HTTP/1.1" 200 -
[2024-10-16 08:22:29,722.722] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 08:22:30,146.146] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 08:22:34,890.890] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 08:22:34,893.893] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 08:23:01,955.955] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 08:23:02,332.332] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 08:23:06,989.989] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 08:23:06,992.992] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 08:23:24,575.575] _internal.py -> _log line:96 [INFO]: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
[2024-10-16 08:23:24,575.575] _internal.py -> _log line:96 [INFO]: [33mPress CTRL+C to quit[0m
[2024-10-16 08:23:24,576.576] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 08:23:29,175.175] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 08:23:29,178.178] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 08:23:29,391.391] base.py -> _exec_single_context line:1903 [INFO]: select pg_catalog.version()
[2024-10-16 08:23:29,391.391] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:23:29,392.392] base.py -> _exec_single_context line:1903 [INFO]: select current_schema()
[2024-10-16 08:23:29,393.393] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:23:29,394.394] base.py -> _exec_single_context line:1903 [INFO]: show standard_conforming_strings
[2024-10-16 08:23:29,394.394] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:23:29,395.395] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 08:23:29,397.397] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 08:23:29,397.397] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00022s] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 08:23:29,401.401] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 08:23:29,401.401] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00025s] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 08:23:29,402.402] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 08:23:30,726.726] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 08:23:30] "GET /conversions?query=广州的天气怎么样？ HTTP/1.1" 200 -
[2024-10-16 08:24:46,641.641] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 08:24:46,997.997] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 08:24:51,909.909] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 08:24:51,912.912] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 08:24:54,526.526] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 08:24:54,862.862] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 08:25:01,861.861] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 08:25:01,864.864] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 08:25:02,311.311] base.py -> _exec_single_context line:1903 [INFO]: select pg_catalog.version()
[2024-10-16 08:25:02,311.311] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:25:02,312.312] base.py -> _exec_single_context line:1903 [INFO]: select current_schema()
[2024-10-16 08:25:02,312.312] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:25:02,313.313] base.py -> _exec_single_context line:1903 [INFO]: show standard_conforming_strings
[2024-10-16 08:25:02,313.313] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:25:02,313.313] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 08:25:02,315.315] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 08:25:02,315.315] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00017s] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 08:25:02,318.318] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 08:25:02,318.318] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00021s] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 08:25:02,319.319] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 08:25:03,767.767] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 08:25:03] "GET /conversions?query=广州的天气怎么样？ HTTP/1.1" 200 -
[2024-10-16 08:27:06,093.093] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 08:27:06,500.500] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 08:27:11,178.178] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 08:27:11,182.182] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 08:27:54,771.771] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 08:27:55,157.157] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 08:27:59,710.710] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 08:27:59,714.714] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 08:28:02,003.003] base.py -> _exec_single_context line:1903 [INFO]: select pg_catalog.version()
[2024-10-16 08:28:02,003.003] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:28:02,004.004] base.py -> _exec_single_context line:1903 [INFO]: select current_schema()
[2024-10-16 08:28:02,004.004] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:28:02,005.005] base.py -> _exec_single_context line:1903 [INFO]: show standard_conforming_strings
[2024-10-16 08:28:02,005.005] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:28:02,006.006] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 08:28:02,007.007] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 08:28:02,007.007] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00017s] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 08:28:02,010.010] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 08:28:02,010.010] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00022s] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 08:28:02,011.011] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 08:28:03,332.332] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 08:28:03] "GET /conversions?query=广州的天气怎么样？ HTTP/1.1" 200 -
[2024-10-16 08:28:40,498.498] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 08:28:40,849.849] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 08:28:45,733.733] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 08:28:45,736.736] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 08:28:45,946.946] base.py -> _exec_single_context line:1903 [INFO]: select pg_catalog.version()
[2024-10-16 08:28:45,946.946] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:28:45,947.947] base.py -> _exec_single_context line:1903 [INFO]: select current_schema()
[2024-10-16 08:28:45,947.947] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:28:45,948.948] base.py -> _exec_single_context line:1903 [INFO]: show standard_conforming_strings
[2024-10-16 08:28:45,948.948] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:28:45,949.949] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 08:28:45,951.951] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 08:28:45,951.951] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00027s] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 08:28:45,954.954] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 08:28:45,954.954] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00018s] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 08:28:45,955.955] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 08:28:51,893.893] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 08:28:51] "GET /conversions?query=广州的天气怎么样？ HTTP/1.1" 200 -
[2024-10-16 08:29:12,410.410] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 08:29:12,411.411] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 08:29:12,411.411] base.py -> _exec_single_context line:1908 [INFO]: [cached since 26.46s ago] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 08:29:12,412.412] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 08:29:12,412.412] base.py -> _exec_single_context line:1908 [INFO]: [cached since 26.46s ago] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 08:29:12,413.413] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 08:29:13,262.262] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 08:29:13] "GET /conversions?query=广州的天气怎么样？ HTTP/1.1" 200 -
[2024-10-16 08:29:17,916.916] http.py -> _register_error_handler line:66 [ERROR]: An error occurred: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
Traceback (most recent call last):
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 844, in dispatch_request
    self.raise_routing_exception(req)
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 453, in raise_routing_exception
    raise request.routing_exception  # type: ignore[misc]
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\ctx.py", line 362, in match_request
    result = self.url_adapter.match(return_rule=True)  # type: ignore
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\werkzeug\routing\map.py", line 624, in match
    raise NotFound() from None
werkzeug.exceptions.NotFound: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
[2024-10-16 08:29:17,931.931] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 08:29:17] "[35m[1mGET /favicon.ico HTTP/1.1[0m" 500 -
[2024-10-16 08:29:39,857.857] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 08:29:40,234.234] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 08:29:45,134.134] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 08:29:45,138.138] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 08:29:46,688.688] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 08:29:47,071.071] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 08:29:51,693.693] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 08:29:51,695.695] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 08:29:51,913.913] base.py -> _exec_single_context line:1903 [INFO]: select pg_catalog.version()
[2024-10-16 08:29:51,913.913] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:29:51,914.914] base.py -> _exec_single_context line:1903 [INFO]: select current_schema()
[2024-10-16 08:29:51,914.914] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:29:51,914.914] base.py -> _exec_single_context line:1903 [INFO]: show standard_conforming_strings
[2024-10-16 08:29:51,914.914] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 08:29:51,915.915] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 08:29:51,917.917] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 08:29:51,917.917] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00021s] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 08:29:51,921.921] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 08:29:51,921.921] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00025s] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 08:29:51,922.922] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 08:29:53,209.209] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 08:29:53] "GET /conversions?query=广州的天气怎么样？ HTTP/1.1" 200 -
[2024-10-16 08:29:57,003.003] http.py -> _register_error_handler line:66 [ERROR]: An error occurred: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
Traceback (most recent call last):
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 844, in dispatch_request
    self.raise_routing_exception(req)
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 453, in raise_routing_exception
    raise request.routing_exception  # type: ignore[misc]
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\ctx.py", line 362, in match_request
    result = self.url_adapter.match(return_rule=True)  # type: ignore
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\werkzeug\routing\map.py", line 624, in match
    raise NotFound() from None
werkzeug.exceptions.NotFound: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
[2024-10-16 08:29:57,037.037] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 08:29:57] "[35m[1mGET /favicon.ico HTTP/1.1[0m" 500 -
[2024-10-16 10:39:10,702.702] http.py -> _register_error_handler line:66 [ERROR]: An error occurred: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
Traceback (most recent call last):
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 844, in dispatch_request
    self.raise_routing_exception(req)
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 453, in raise_routing_exception
    raise request.routing_exception  # type: ignore[misc]
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\ctx.py", line 362, in match_request
    result = self.url_adapter.match(return_rule=True)  # type: ignore
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\werkzeug\routing\map.py", line 624, in match
    raise NotFound() from None
werkzeug.exceptions.NotFound: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
[2024-10-16 10:39:10,713.713] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 10:39:10] "[35m[1mGET /conversations?query=能看下2024年北京半程马拉松的前3名成绩是多少么？ HTTP/1.1[0m" 500 -
[2024-10-16 10:39:10,814.814] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 10:39:10] "[36mGET /conversations?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 304 -
[2024-10-16 10:39:10,954.954] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 10:39:10] "[36mGET /conversations?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 304 -
[2024-10-16 10:39:11,278.278] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 10:39:11] "[36mGET /conversations?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 -
[2024-10-16 10:39:37,158.158] base.py -> get_connection line:842 [INFO]: Connection <connection object at 0x000002A81A31D580; dsn: 'user=postgres password=xxx dbname=llmops host=192.168.110.240 port=5432 client_encoding=utf8', closed: 0> exceeded timeout; recycling
[2024-10-16 10:39:37,170.170] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 10:39:37,171.171] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 10:39:37,171.171] base.py -> _exec_single_context line:1908 [INFO]: [cached since 7785s ago] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 10:39:37,174.174] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 10:39:37,175.175] base.py -> _exec_single_context line:1908 [INFO]: [cached since 7785s ago] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 10:39:37,176.176] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 10:39:38,170.170] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 10:39:38] "GET /conversions?query=能看下2024年北京半程马拉松的前3名成绩是多少么？ HTTP/1.1" 200 -
[2024-10-16 10:40:25,656.656] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 10:40:26,052.052] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 10:40:31,052.052] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 10:40:31,055.055] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 10:40:34,902.902] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 10:40:35,298.298] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 10:40:39,707.707] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 10:40:39,710.710] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 10:40:39,915.915] base.py -> _exec_single_context line:1903 [INFO]: select pg_catalog.version()
[2024-10-16 10:40:39,915.915] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 10:40:39,915.915] base.py -> _exec_single_context line:1903 [INFO]: select current_schema()
[2024-10-16 10:40:39,915.915] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 10:40:39,915.915] base.py -> _exec_single_context line:1903 [INFO]: show standard_conforming_strings
[2024-10-16 10:40:39,915.915] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 10:40:39,916.916] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 10:40:39,918.918] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 10:40:39,918.918] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00015s] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 10:40:39,921.921] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 10:40:39,921.921] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00017s] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 10:40:39,922.922] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 10:40:41,163.163] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 10:40:41] "GET /conversions?query=能看下2024年北京半程马拉松的前3名成绩是多少么？ HTTP/1.1" 200 -
[2024-10-16 10:40:44,300.300] http.py -> _register_error_handler line:66 [ERROR]: An error occurred: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
Traceback (most recent call last):
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 844, in dispatch_request
    self.raise_routing_exception(req)
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 453, in raise_routing_exception
    raise request.routing_exception  # type: ignore[misc]
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\ctx.py", line 362, in match_request
    result = self.url_adapter.match(return_rule=True)  # type: ignore
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\werkzeug\routing\map.py", line 624, in match
    raise NotFound() from None
werkzeug.exceptions.NotFound: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
[2024-10-16 10:40:44,312.312] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 10:40:44] "[35m[1mGET /favicon.ico HTTP/1.1[0m" 500 -
[2024-10-16 10:40:56,758.758] http.py -> _register_error_handler line:66 [ERROR]: An error occurred: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
Traceback (most recent call last):
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 844, in dispatch_request
    self.raise_routing_exception(req)
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 453, in raise_routing_exception
    raise request.routing_exception  # type: ignore[misc]
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\ctx.py", line 362, in match_request
    result = self.url_adapter.match(return_rule=True)  # type: ignore
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\werkzeug\routing\map.py", line 624, in match
    raise NotFound() from None
werkzeug.exceptions.NotFound: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
[2024-10-16 10:40:56,774.774] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 10:40:56] "[35m[1mGET /favicon.ico HTTP/1.1[0m" 500 -
[2024-10-16 11:50:01,554.554] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\model\\conversation.py', reloading
[2024-10-16 11:50:01,960.960] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 11:50:07,194.194] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 11:50:07,197.197] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 15:29:53,818.818] _internal.py -> _log line:96 [INFO]: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
[2024-10-16 15:29:53,819.819] _internal.py -> _log line:96 [INFO]: [33mPress CTRL+C to quit[0m
[2024-10-16 15:29:53,820.820] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 15:29:58,338.338] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 15:29:58,341.341] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 15:29:58,579.579] base.py -> _exec_single_context line:1903 [INFO]: select pg_catalog.version()
[2024-10-16 15:29:58,579.579] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 15:29:58,580.580] base.py -> _exec_single_context line:1903 [INFO]: select current_schema()
[2024-10-16 15:29:58,581.581] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 15:29:58,581.581] base.py -> _exec_single_context line:1903 [INFO]: show standard_conforming_strings
[2024-10-16 15:29:58,581.581] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 15:29:58,582.582] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 15:29:58,584.584] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 15:29:58,584.584] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00020s] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 15:29:58,662.662] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 15:29:58,662.662] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00021s] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 15:29:58,789.789] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 15:30:00,474.474] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 15:30:00] "GET /conversions?query=广州的天气怎么样？ HTTP/1.1" 200 -
[2024-10-16 15:30:10,221.221] http.py -> _register_error_handler line:66 [ERROR]: An error occurred: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
Traceback (most recent call last):
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 844, in dispatch_request
    self.raise_routing_exception(req)
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 453, in raise_routing_exception
    raise request.routing_exception  # type: ignore[misc]
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\ctx.py", line 362, in match_request
    result = self.url_adapter.match(return_rule=True)  # type: ignore
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\werkzeug\routing\map.py", line 624, in match
    raise NotFound() from None
werkzeug.exceptions.NotFound: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
[2024-10-16 15:30:10,242.242] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 15:30:10] "[35m[1mGET /favicon.ico HTTP/1.1[0m" 500 -
[2024-10-16 15:30:11,700.700] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 15:30:11,701.701] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 15:30:11,701.701] base.py -> _exec_single_context line:1908 [INFO]: [cached since 13.12s ago] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 15:30:11,703.703] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 15:30:11,703.703] base.py -> _exec_single_context line:1908 [INFO]: [cached since 13.04s ago] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 15:30:11,705.705] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 15:30:13,204.204] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 15:30:13] "GET /conversions?query=广州的天气怎么样？ HTTP/1.1" 200 -
[2024-10-16 15:30:19,258.258] http.py -> _register_error_handler line:66 [ERROR]: An error occurred: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
Traceback (most recent call last):
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 844, in dispatch_request
    self.raise_routing_exception(req)
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 453, in raise_routing_exception
    raise request.routing_exception  # type: ignore[misc]
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\ctx.py", line 362, in match_request
    result = self.url_adapter.match(return_rule=True)  # type: ignore
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\werkzeug\routing\map.py", line 624, in match
    raise NotFound() from None
werkzeug.exceptions.NotFound: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
[2024-10-16 15:30:19,270.270] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 15:30:19] "[35m[1mGET /favicon.ico HTTP/1.1[0m" 500 -
[2024-10-16 15:31:19,854.854] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 15:31:19,855.855] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 15:31:19,855.855] base.py -> _exec_single_context line:1908 [INFO]: [cached since 81.27s ago] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 15:31:19,857.857] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 15:31:19,857.857] base.py -> _exec_single_context line:1908 [INFO]: [cached since 81.19s ago] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 15:31:19,858.858] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 15:31:43,126.126] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 15:31:43,127.127] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 15:31:43,127.127] base.py -> _exec_single_context line:1908 [INFO]: [cached since 104.5s ago] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 15:31:43,128.128] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 15:31:43,129.129] base.py -> _exec_single_context line:1908 [INFO]: [cached since 104.5s ago] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 15:31:43,129.129] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 15:31:51,782.782] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 15:31:51,782.782] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 15:31:51,783.783] base.py -> _exec_single_context line:1908 [INFO]: [cached since 113.2s ago] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 15:31:51,784.784] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 15:31:51,784.784] base.py -> _exec_single_context line:1908 [INFO]: [cached since 113.1s ago] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 15:31:51,785.785] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 15:31:52,690.690] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 15:31:52] "GET /conversions?query=广州的天气怎么样？ HTTP/1.1" 200 -
[2024-10-16 15:31:56,419.419] http.py -> _register_error_handler line:66 [ERROR]: An error occurred: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
Traceback (most recent call last):
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 844, in dispatch_request
    self.raise_routing_exception(req)
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 453, in raise_routing_exception
    raise request.routing_exception  # type: ignore[misc]
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\ctx.py", line 362, in match_request
    result = self.url_adapter.match(return_rule=True)  # type: ignore
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\werkzeug\routing\map.py", line 624, in match
    raise NotFound() from None
werkzeug.exceptions.NotFound: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
[2024-10-16 15:31:56,444.444] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 15:31:56] "[35m[1mGET /favicon.ico HTTP/1.1[0m" 500 -
[2024-10-16 15:32:13,076.076] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 15:32:13,078.078] base.py -> _exec_single_context line:1903 [INFO]: SELECT dataset.id AS dataset_id, dataset.account_id AS dataset_account_id, dataset.name AS dataset_name, dataset.icon AS dataset_icon, dataset.description AS dataset_description, dataset.updated_at AS dataset_updated_at, dataset.created_at AS dataset_created_at 
FROM dataset 
WHERE dataset.account_id = %(account_id_1)s::UUID ORDER BY dataset.created_at DESC 
 LIMIT %(param_1)s OFFSET %(param_2)s
[2024-10-16 15:32:13,078.078] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00029s] {'account_id_1': '46db30d1-3199-4e79-a0cd-abf12fa6858f', 'param_1': 20, 'param_2': 0}
[2024-10-16 15:32:13,109.109] base.py -> _exec_single_context line:1903 [INFO]: SELECT count(*) AS count_1 
FROM (SELECT dataset.id AS id, dataset.account_id AS account_id, dataset.name AS name, dataset.icon AS icon, dataset.description AS description, dataset.updated_at AS updated_at, dataset.created_at AS created_at 
FROM dataset 
WHERE dataset.account_id = %(account_id_1)s::UUID) AS anon_1
[2024-10-16 15:32:13,109.109] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00026s] {'account_id_1': '46db30d1-3199-4e79-a0cd-abf12fa6858f'}
[2024-10-16 15:32:13,111.111] base.py -> _exec_single_context line:1903 [INFO]: SELECT count(document.id) AS count_1 
FROM document 
WHERE document.dataset_id = %(dataset_id_1)s::UUID
[2024-10-16 15:32:13,112.112] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00025s] {'dataset_id_1': UUID('f3f28f75-8e60-4eba-b6df-4d1b390bbd89')}
[2024-10-16 15:32:13,252.252] base.py -> _exec_single_context line:1903 [INFO]: SELECT count(app_dataset_join.id) AS count_1 
FROM app_dataset_join 
WHERE app_dataset_join.dataset_id = %(dataset_id_1)s::UUID
[2024-10-16 15:32:13,252.252] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00028s] {'dataset_id_1': UUID('f3f28f75-8e60-4eba-b6df-4d1b390bbd89')}
[2024-10-16 15:32:13,275.275] base.py -> _exec_single_context line:1903 [INFO]: SELECT coalesce(sum(document.character_count), %(coalesce_2)s) AS coalesce_1 
FROM document 
WHERE document.dataset_id = %(dataset_id_1)s::UUID
[2024-10-16 15:32:13,276.276] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00028s] {'coalesce_2': 0, 'dataset_id_1': UUID('f3f28f75-8e60-4eba-b6df-4d1b390bbd89')}
[2024-10-16 15:32:13,277.277] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 15:32:13,278.278] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 15:32:13] "GET /datasets?search_word=&current_page=1&page_size=20 HTTP/1.1" 200 -
[2024-10-16 15:32:20,334.334] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 15:32:20,335.335] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 15:32:20,335.335] base.py -> _exec_single_context line:1908 [INFO]: [cached since 141.8s ago] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 15:32:20,336.336] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 15:32:20,336.336] base.py -> _exec_single_context line:1908 [INFO]: [cached since 141.7s ago] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 15:32:20,337.337] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 15:32:40,745.745] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 15:32:40] "GET /conversions?query=广州的天气怎么样？ HTTP/1.1" 200 -
[2024-10-16 15:33:25,326.326] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 15:33:25] "GET /conversions?query=广州的天气怎么样？ HTTP/1.1" 200 -
[2024-10-16 15:33:28,787.787] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 15:33:28,788.788] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 15:33:28,788.788] base.py -> _exec_single_context line:1908 [INFO]: [cached since 210.2s ago] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 15:33:28,790.790] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 15:33:28,790.790] base.py -> _exec_single_context line:1908 [INFO]: [cached since 210.1s ago] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 15:33:28,791.791] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 15:33:29,526.526] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 15:33:29] "GET /conversions?query=你好，你是？ HTTP/1.1" 200 -
[2024-10-16 15:33:57,767.767] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 15:33:57,767.767] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 15:33:57,767.767] base.py -> _exec_single_context line:1908 [INFO]: [cached since 239.2s ago] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 15:33:57,769.769] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 15:33:57,769.769] base.py -> _exec_single_context line:1908 [INFO]: [cached since 239.1s ago] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 15:33:57,770.770] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 15:33:59,700.700] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 15:33:59] "GET /conversions?query=帮我查下2024年芝加哥马拉松的冠军是谁，成绩是多少？ HTTP/1.1" 200 -
[2024-10-16 15:34:31,891.891] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 15:34:31] "GET /conversions?query=广州的天气怎么样？ HTTP/1.1" 200 -
[2024-10-16 15:35:01,682.682] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 15:35:01,682.682] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 15:35:01,682.682] base.py -> _exec_single_context line:1908 [INFO]: [cached since 303.1s ago] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 15:35:01,684.684] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 15:35:01,684.684] base.py -> _exec_single_context line:1908 [INFO]: [cached since 303s ago] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 15:35:01,685.685] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 15:35:02,567.567] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 15:35:02] "GET /conversions?query=帮我查下2024年芝加哥马拉松的冠军是谁，成绩是多少？ HTTP/1.1" 200 -
[2024-10-16 15:35:33,236.236] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 15:35:33,237.237] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 15:35:33,237.237] base.py -> _exec_single_context line:1908 [INFO]: [cached since 334.7s ago] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 15:35:33,238.238] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 15:35:33,238.238] base.py -> _exec_single_context line:1908 [INFO]: [cached since 334.6s ago] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 15:35:33,239.239] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 15:35:34,107.107] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 15:35:34] "GET /conversions?query=帮我查下2024年芝加哥马拉松的冠军是谁，成绩是多少？ HTTP/1.1" 200 -
[2024-10-16 15:35:47,045.045] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 15:35:47,046.046] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 15:35:47,046.046] base.py -> _exec_single_context line:1908 [INFO]: [cached since 348.5s ago] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 15:35:47,047.047] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 15:35:47,048.048] base.py -> _exec_single_context line:1908 [INFO]: [cached since 348.4s ago] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 15:35:47,049.049] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 15:35:48,050.050] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 15:35:48] "GET /conversions?query=帮我查下2024年芝加哥马拉松的冠军是谁，成绩是多少？ HTTP/1.1" 200 -
[2024-10-16 15:35:58,394.394] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\pkg\\response\\response.py', reloading
[2024-10-16 15:36:10,816.816] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 15:36:15,226.226] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 15:36:15,229.229] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 15:36:15,429.429] base.py -> _exec_single_context line:1903 [INFO]: select pg_catalog.version()
[2024-10-16 15:36:15,429.429] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 15:36:15,430.430] base.py -> _exec_single_context line:1903 [INFO]: select current_schema()
[2024-10-16 15:36:15,430.430] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 15:36:15,431.431] base.py -> _exec_single_context line:1903 [INFO]: show standard_conforming_strings
[2024-10-16 15:36:15,431.431] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 15:36:15,432.432] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 15:36:15,433.433] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 15:36:15,434.434] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00017s] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 15:36:15,437.437] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 15:36:15,437.437] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00018s] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 15:36:15,438.438] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 15:36:16,604.604] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 15:36:16] "GET /conversions?query=帮我查下2024年芝加哥马拉松的冠军是谁，成绩是多少？ HTTP/1.1" 200 -
[2024-10-16 15:37:54,319.319] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 15:37:54,319.319] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 15:37:54,320.320] base.py -> _exec_single_context line:1908 [INFO]: [cached since 98.89s ago] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 15:37:54,321.321] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 15:37:54,321.321] base.py -> _exec_single_context line:1908 [INFO]: [cached since 98.88s ago] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 15:37:54,322.322] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 15:37:55,165.165] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 15:37:55] "GET /conversions?query=广州最近的天气怎么样 HTTP/1.1" 200 -
[2024-10-16 15:38:24,110.110] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 15:38:24,110.110] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 15:38:24,111.111] base.py -> _exec_single_context line:1908 [INFO]: [cached since 128.7s ago] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 15:38:24,112.112] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 15:38:24,112.112] base.py -> _exec_single_context line:1908 [INFO]: [cached since 128.7s ago] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 15:38:24,112.112] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 15:38:24,994.994] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 15:38:24] "GET /conversions?query=广州最近的天气怎么样 HTTP/1.1" 200 -
[2024-10-16 15:38:30,560.560] http.py -> _register_error_handler line:66 [ERROR]: An error occurred: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
Traceback (most recent call last):
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 844, in dispatch_request
    self.raise_routing_exception(req)
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 453, in raise_routing_exception
    raise request.routing_exception  # type: ignore[misc]
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\ctx.py", line 362, in match_request
    result = self.url_adapter.match(return_rule=True)  # type: ignore
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\werkzeug\routing\map.py", line 624, in match
    raise NotFound() from None
werkzeug.exceptions.NotFound: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
[2024-10-16 15:38:30,571.571] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 15:38:30] "[35m[1mGET /favicon.ico HTTP/1.1[0m" 500 -
[2024-10-16 15:38:58,684.684] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 15:38:58,685.685] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 15:38:58,685.685] base.py -> _exec_single_context line:1908 [INFO]: [cached since 163.3s ago] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 15:38:58,686.686] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 15:38:58,686.686] base.py -> _exec_single_context line:1908 [INFO]: [cached since 163.2s ago] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 15:38:58,687.687] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 15:38:59,550.550] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 15:38:59] "GET /conversions?query=深圳最近的天气怎么样 HTTP/1.1" 200 -
[2024-10-16 19:54:04,693.693] base.py -> get_connection line:842 [INFO]: Connection <connection object at 0x0000017F3A5CD8B0; dsn: 'user=postgres password=xxx dbname=llmops host=192.168.110.240 port=5432 client_encoding=utf8', closed: 0> exceeded timeout; recycling
[2024-10-16 19:54:04,715.715] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 19:54:04,718.718] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 19:54:04,719.719] base.py -> _exec_single_context line:1908 [INFO]: [cached since 1.547e+04s ago] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 19:54:04,728.728] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 19:54:04,728.728] base.py -> _exec_single_context line:1908 [INFO]: [cached since 1.547e+04s ago] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 19:54:04,730.730] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 19:54:05,919.919] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 19:54:05] "GET /conversions?query=广州的天气怎么样？ HTTP/1.1" 200 -
[2024-10-16 19:54:17,453.453] http.py -> _register_error_handler line:66 [ERROR]: An error occurred: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
Traceback (most recent call last):
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 844, in dispatch_request
    self.raise_routing_exception(req)
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 453, in raise_routing_exception
    raise request.routing_exception  # type: ignore[misc]
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\ctx.py", line 362, in match_request
    result = self.url_adapter.match(return_rule=True)  # type: ignore
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\werkzeug\routing\map.py", line 624, in match
    raise NotFound() from None
werkzeug.exceptions.NotFound: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
[2024-10-16 19:54:17,479.479] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 19:54:17] "[35m[1mGET /favicon.ico HTTP/1.1[0m" 500 -
[2024-10-16 20:11:17,483.483] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\model\\conversation.py', reloading
[2024-10-16 20:11:17,957.957] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 20:11:22,698.698] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 20:11:22,701.701] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 20:36:48,496.496] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 20:36:48,822.822] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 20:36:53,717.717] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 20:36:53,720.720] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 20:36:53,911.911] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 20:36:53] "GET /conversions?query=广州的天气怎么样？ HTTP/1.1" 200 -
[2024-10-16 20:36:53,926.926] base.py -> _exec_single_context line:1903 [INFO]: select pg_catalog.version()
[2024-10-16 20:36:53,926.926] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 20:36:53,928.928] base.py -> _exec_single_context line:1903 [INFO]: select current_schema()
[2024-10-16 20:36:53,928.928] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 20:36:53,929.929] base.py -> _exec_single_context line:1903 [INFO]: show standard_conforming_strings
[2024-10-16 20:36:53,929.929] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 20:36:53,931.931] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 20:36:53,936.936] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 20:36:53,936.936] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00041s] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 20:36:53,939.939] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 20:36:53,939.939] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00019s] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 20:36:53,940.940] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 20:36:53,971.971] http.py -> _register_error_handler line:66 [ERROR]: An error occurred: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
Traceback (most recent call last):
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 844, in dispatch_request
    self.raise_routing_exception(req)
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 453, in raise_routing_exception
    raise request.routing_exception  # type: ignore[misc]
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\ctx.py", line 362, in match_request
    result = self.url_adapter.match(return_rule=True)  # type: ignore
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\werkzeug\routing\map.py", line 624, in match
    raise NotFound() from None
werkzeug.exceptions.NotFound: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
[2024-10-16 20:36:53,986.986] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 20:36:53] "[35m[1mGET /favicon.ico HTTP/1.1[0m" 500 -
[2024-10-16 20:36:58,797.797] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\conversation_handler.py', reloading
[2024-10-16 20:36:59,122.122] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 20:37:03,961.961] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 20:37:03,964.964] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 20:38:40,661.661] base.py -> _exec_single_context line:1903 [INFO]: select pg_catalog.version()
[2024-10-16 20:38:40,661.661] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 20:38:40,661.661] base.py -> _exec_single_context line:1903 [INFO]: select current_schema()
[2024-10-16 20:38:40,662.662] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 20:38:40,662.662] base.py -> _exec_single_context line:1903 [INFO]: show standard_conforming_strings
[2024-10-16 20:38:40,662.662] base.py -> _exec_single_context line:1908 [INFO]: [raw sql] {}
[2024-10-16 20:38:40,663.663] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 20:38:40,666.666] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 20:38:40,666.666] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00026s] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 20:38:40,670.670] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 20:38:40,671.671] base.py -> _exec_single_context line:1908 [INFO]: [generated in 0.00024s] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 20:38:40,672.672] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 20:38:42,642.642] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 20:38:42] "GET /conversions?query=广州的天气怎么样？ HTTP/1.1" 200 -
[2024-10-16 20:38:49,260.260] http.py -> _register_error_handler line:66 [ERROR]: An error occurred: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
Traceback (most recent call last):
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 844, in dispatch_request
    self.raise_routing_exception(req)
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\app.py", line 453, in raise_routing_exception
    raise request.routing_exception  # type: ignore[misc]
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\flask\ctx.py", line 362, in match_request
    result = self.url_adapter.match(return_rule=True)  # type: ignore
  File "D:\Code\llmops\llmops-api\venv\lib\site-packages\werkzeug\routing\map.py", line 624, in match
    raise NotFound() from None
werkzeug.exceptions.NotFound: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
[2024-10-16 20:38:49,290.290] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 20:38:49] "[35m[1mGET /favicon.ico HTTP/1.1[0m" 500 -
[2024-10-16 21:00:11,547.547] base.py -> _begin_impl line:1103 [INFO]: BEGIN (implicit)
[2024-10-16 21:00:11,548.548] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool.id AS api_tool_id, api_tool.account_id AS api_tool_account_id, api_tool.provider_id AS api_tool_provider_id, api_tool.name AS api_tool_name, api_tool.description AS api_tool_description, api_tool.url AS api_tool_url, api_tool.method AS api_tool_method, api_tool.parameters AS api_tool_parameters, api_tool.updated_at AS api_tool_updated_at, api_tool.created_at AS api_tool_created_at 
FROM api_tool 
WHERE api_tool.provider_id = %(provider_id_1)s::UUID AND api_tool.name = %(name_1)s
[2024-10-16 21:00:11,548.548] base.py -> _exec_single_context line:1908 [INFO]: [cached since 1291s ago] {'provider_id_1': 'd72bb9d7-8794-4caf-bd60-1f992c537065', 'name_1': 'YoudaoSuggest'}
[2024-10-16 21:00:11,549.549] base.py -> _exec_single_context line:1903 [INFO]: SELECT api_tool_provider.id AS api_tool_provider_id, api_tool_provider.account_id AS api_tool_provider_account_id, api_tool_provider.name AS api_tool_provider_name, api_tool_provider.icon AS api_tool_provider_icon, api_tool_provider.description AS api_tool_provider_description, api_tool_provider.openapi_schema AS api_tool_provider_openapi_schema, api_tool_provider.headers AS api_tool_provider_headers, api_tool_provider.updated_at AS api_tool_provider_updated_at, api_tool_provider.created_at AS api_tool_provider_created_at 
FROM api_tool_provider 
WHERE api_tool_provider.id = %(pk_1)s::UUID
[2024-10-16 21:00:11,549.549] base.py -> _exec_single_context line:1908 [INFO]: [cached since 1291s ago] {'pk_1': UUID('d72bb9d7-8794-4caf-bd60-1f992c537065')}
[2024-10-16 21:00:11,550.550] base.py -> _rollback_impl line:1129 [INFO]: ROLLBACK
[2024-10-16 21:00:13,498.498] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 21:00:13] "GET /conversions?query=深圳最近的天气怎么样 HTTP/1.1" 200 -
[2024-10-16 22:13:51,931.931] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:13:52,368.368] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:13:57,012.012] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:13:57,014.014] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:14:18,145.145] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:14:18,444.444] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:14:22,864.864] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:14:22,867.867] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:15:13,310.310] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:15:13,617.617] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:15:18,337.337] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:15:18,339.339] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:15:54,441.441] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:15:54,804.804] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:18:01,626.626] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:18:06,185.185] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:18:06,187.187] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:18:49,456.456] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:18:49,807.807] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:18:54,381.381] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:18:54,384.384] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:20:28,602.602] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:20:28,917.917] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:20:33,476.476] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:20:33,482.482] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:21:17,981.981] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:21:18,387.387] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:21:22,858.858] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:21:22,861.861] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:21:57,007.007] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:21:57,465.465] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:22:02,145.145] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:22:02,148.148] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:23:22,617.617] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:23:22,924.924] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:23:27,486.486] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:23:27,489.489] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:25:37,104.104] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:25:37,468.468] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:25:42,406.406] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:25:42,409.409] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:25:52,198.198] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:25:52,528.528] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:25:56,988.988] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:25:56,990.990] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:26:45,025.025] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:26:45,404.404] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:26:49,994.994] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:26:49,998.998] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:27:29,679.679] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:27:29,997.997] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:27:51,529.529] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:27:55,910.910] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:27:55,912.912] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:28:35,894.894] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:28:36,275.275] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:28:40,738.738] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:28:40,741.741] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:29:21,707.707] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:29:22,000.000] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:30:47,457.457] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:30:52,434.434] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:30:52,437.437] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:31:09,095.095] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:31:09,476.476] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:31:15,853.853] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:31:15,858.858] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:32:21,321.321] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:32:21,642.642] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:32:26,594.594] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:32:26,597.597] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:32:29,190.190] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:32:29,625.625] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:32:34,056.056] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:32:34,058.058] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:32:46,099.099] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:32:46,430.430] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:32:51,457.457] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:32:51,460.460] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:33:19,652.652] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:33:19,999.999] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:33:24,463.463] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:33:24,466.466] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:33:51,485.485] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:33:51,842.842] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:33:56,748.748] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:33:56,751.751] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:35:43,230.230] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:35:43,600.600] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:35:48,165.165] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:35:48,168.168] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:36:27,143.143] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:36:27,462.462] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:36:32,138.138] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:36:32,142.142] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:36:38,190.190] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:36:38,506.506] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:36:43,180.180] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:36:43,184.184] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:37:20,055.055] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:37:20,391.391] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:37:25,087.087] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:37:25,090.090] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:37:28,393.393] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 22:37:28] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 22:38:46,561.561] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:38:46,894.894] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:38:51,485.485] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:38:51,487.487] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:38:53,378.378] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 22:38:53] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 22:39:32,602.602] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:39:32,915.915] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:39:37,355.355] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:39:37,357.357] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:39:41,865.865] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 22:39:41] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 22:39:55,068.068] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:39:55,401.401] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:40:00,508.508] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:40:00,509.509] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:40:00,776.776] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:40:01,097.097] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:40:05,734.734] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:40:05,737.737] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:40:09,990.990] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 22:40:09] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 22:41:19,096.096] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:41:19,437.437] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:41:23,956.956] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:41:23,958.958] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:41:27,755.755] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 22:41:27] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 22:41:55,765.765] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:41:56,106.106] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:42:00,758.758] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:42:00,762.762] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:42:06,801.801] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:42:07,088.088] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:42:11,728.728] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:42:11,732.732] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:42:32,842.842] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 22:42:32] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 22:42:51,387.387] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 22:42:51] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 22:43:18,560.560] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\router\\router.py', reloading
[2024-10-16 22:43:18,911.911] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:43:24,541.541] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:43:24,544.544] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:43:24,703.703] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 22:43:24] "GET /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 22:43:43,139.139] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 22:43:43] "GET /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug?query=广州最近的天气怎样 HTTP/1.1" 200 -
[2024-10-16 22:43:54,531.531] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\router\\router.py', reloading
[2024-10-16 22:43:54,828.828] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:43:59,320.320] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:43:59,323.323] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:44:05,094.094] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 22:44:05] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 22:44:23,055.055] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:44:23,437.437] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:44:28,166.166] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:44:28,169.169] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:44:30,697.697] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 22:44:30] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 22:44:53,981.981] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:44:54,383.383] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:44:59,204.204] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:44:59,207.207] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:45:02,373.373] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 22:45:02] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 22:45:16,905.905] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:45:17,206.206] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:45:21,731.731] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:45:21,734.734] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:45:24,369.369] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:45:24,763.763] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:45:30,831.831] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:45:30,834.834] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:45:32,556.556] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 22:45:32] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 22:46:10,773.773] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 22:46:10] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 22:46:36,087.087] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:47:12,969.969] _internal.py -> _log line:96 [INFO]: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
[2024-10-16 22:47:12,969.969] _internal.py -> _log line:96 [INFO]: [33mPress CTRL+C to quit[0m
[2024-10-16 22:47:12,970.970] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:47:17,518.518] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:47:17,520.520] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:47:19,225.225] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 22:47:19] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 22:50:22,371.371] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:50:22,676.676] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:50:27,747.747] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:50:27,750.750] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:51:16,315.315] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:51:16,697.697] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:51:21,647.647] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:51:21,649.649] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:51:34,803.803] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:51:35,187.187] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:51:39,835.835] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:51:39,840.840] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:51:45,366.366] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 22:51:45] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 22:51:58,524.524] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 22:51:58] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 22:52:34,059.059] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 22:52:34] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 22:52:47,938.938] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 22:52:47] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 22:53:38,387.387] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 22:53:38,745.745] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 22:53:43,253.253] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 22:53:43,256.256] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 22:53:48,661.661] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 22:53:48] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 22:57:07,375.375] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 22:57:07] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 22:57:24,983.983] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 22:57:24] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 23:01:14,997.997] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 23:01:15,449.449] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:01:19,945.945] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 23:01:19,948.948] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 23:01:41,116.116] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 23:01:41,430.430] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:01:45,972.972] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 23:01:45,975.975] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 23:01:53,238.238] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 23:01:53] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 23:02:19,870.870] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 23:02:20,182.182] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:02:24,991.991] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 23:02:24,993.993] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 23:04:03,682.682] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 23:04:04,014.014] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:04:08,605.605] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 23:04:08,608.608] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 23:04:28,721.721] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 23:04:29,037.037] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:04:33,923.923] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 23:04:33,926.926] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 23:05:38,163.163] _internal.py -> _log line:96 [INFO]: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
[2024-10-16 23:05:38,164.164] _internal.py -> _log line:96 [INFO]: [33mPress CTRL+C to quit[0m
[2024-10-16 23:05:38,166.166] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:05:42,608.608] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 23:05:42,611.611] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 23:07:03,180.180] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 23:07:03,581.581] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:07:08,064.064] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 23:07:08,065.065] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 23:07:12,317.317] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 23:07:12] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 23:14:28,420.420] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\千帆测试.py', reloading
[2024-10-16 23:14:28,812.812] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:14:33,419.419] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 23:14:33,422.422] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 23:14:42,014.014] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\千帆测试.py', reloading
[2024-10-16 23:14:42,316.316] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:14:46,875.875] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 23:14:46,878.878] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 23:15:33,592.592] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\千帆测试.py', reloading
[2024-10-16 23:15:33,909.909] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:15:38,602.602] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 23:15:38,605.605] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 23:16:52,708.708] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\千帆测试.py', reloading
[2024-10-16 23:16:53,014.014] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:16:58,027.027] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 23:16:58,030.030] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 23:16:58,344.344] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\千帆测试.py', reloading
[2024-10-16 23:16:58,665.665] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:17:03,515.515] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 23:17:03,517.517] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 23:18:15,708.708] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 23:18:16,171.171] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:18:20,557.557] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 23:18:20,561.561] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 23:18:51,527.527] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 23:18:51] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 23:19:21,642.642] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 23:19:21] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 23:25:16,198.198] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 23:25:16] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 23:28:58,106.106] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 23:28:58,471.471] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:29:03,067.067] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 23:29:03,069.069] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 23:29:37,368.368] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 23:29:37] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 23:30:29,167.167] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 23:30:29,487.487] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:30:34,156.156] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 23:30:34,159.159] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 23:30:37,431.431] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 23:30:37] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 23:31:42,732.732] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 23:31:42] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 23:31:52,020.020] _internal.py -> _log line:96 [INFO]: 127.0.0.1 - - [16/Oct/2024 23:31:52] "POST /apps/e0d13c78-870b-46df-b2f5-693ae9d5d727/debug HTTP/1.1" 200 -
[2024-10-16 23:35:02,823.823] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 23:35:03,149.149] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:35:08,156.156] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 23:35:08,159.159] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 23:37:20,787.787] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\__init__.py', reloading
[2024-10-16 23:37:21,113.113] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:37:25,686.686] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 23:37:25,688.688] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 23:37:36,382.382] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\router\\router.py', reloading
[2024-10-16 23:37:36,740.740] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:37:45,876.876] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:37:50,478.478] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 23:37:50,481.481] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 23:38:29,114.114] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\pkg\\response\\response.py', reloading
[2024-10-16 23:38:29,456.456] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:38:31,553.553] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:38:49,809.809] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:38:54,413.413] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 23:38:54,416.416] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 23:38:58,262.262] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 23:38:58,594.594] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:39:03,120.120] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 23:39:03,123.123] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 23:48:49,163.163] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 23:48:49,479.479] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:48:53,787.787] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 23:48:53,790.790] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 23:49:23,074.074] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 23:49:23,421.421] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:49:28,117.117] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 23:49:28,119.119] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 23:53:30,505.505] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\pkg\\response\\response.py', reloading
[2024-10-16 23:53:30,882.882] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:53:35,322.322] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 23:53:35,324.324] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 23:53:48,336.336] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\pkg\\response\\__init__.py', reloading
[2024-10-16 23:53:48,668.668] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:53:53,105.105] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 23:53:53,108.108] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 23:55:00,671.671] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 23:55:00,948.948] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:55:05,695.695] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 23:55:05,697.697] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 23:56:49,069.069] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 23:56:49,451.451] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:56:54,009.009] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 23:56:54,012.012] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
[2024-10-16 23:58:59,467.467] _internal.py -> _log line:96 [INFO]:  * Detected change in 'D:\\Code\\llmops\\llmops-api\\internal\\handler\\app_handler.py', reloading
[2024-10-16 23:58:59,811.811] _internal.py -> _log line:96 [INFO]:  * Restarting with stat
[2024-10-16 23:59:04,184.184] _internal.py -> _log line:96 [WARNING]:  * Debugger is active!
[2024-10-16 23:59:04,187.187] _internal.py -> _log line:96 [INFO]:  * Debugger PIN: 102-324-223
